mrblib/haconiwa/runner.rb

```diff
@@ -484,5 +487,26 @@ def persist_namespace(pid, namespace)
         end
       end
     end
+
+    def process_exists?(pid)
+      begin
+        ::Process.kill(0, pid)
+      rescue
+        false
+      end
+    end
+
+    def confirm_existence_pid_file(pid_file)
+      if File.exist?(pid_file) && process_exists?(File.read(pid_file).to_i)
+        raise "PID file #{pid_file} exists. You may be creating the container with existing name #{@base.name}!"
+      else
+        begin
+          File.unlink(pid_file)
```

Comment:
It seems that even if the file `pid_file` does not exist, haconiwa tries to unlink the file!